morecantile/defaults.py

```diff
@@ -33,7 +33,12 @@ def get(self, identifier: str) -> TileMatrixSet:
         if identifier not in self.tms:
             raise InvalidIdentifier(f"Invalid identifier: {identifier}")
 
-        return self.tms[identifier]
+        tms = self.tms[identifier]
+        if isinstance(tms, pathlib.Path):
+            tms = TileMatrixSet.parse_file(tms)
+            self.tms[identifier] = tms
```

Comment:
if TMS is a `Path` object we parse it and store it in the dict so future call won't need to `parse`